lib/round.rb

```diff
@@ -9,49 +9,57 @@ def initialize(deck)
     @deck = deck
     @guesses = []
     @number_correct = 0
-    @current_card = 0
+    @card_position = 0
```

Comment:
This naming better represents how this value is being used, which is as an index position.